config.js

```diff
@@ -21,20 +21,174 @@
  * please contact with::[daniel.moranjimenez at telefonica.com]
  */
 
-var config = {
+var config = {};
```

Comment:
Is config.js documented in the README.md or other .md docs in other agents (JSON, UL)? Should be do the same for Sigfox one?

Reply:
Just config.sigfox.port is documented in README.md. Rest of config.js is common to iiotagent-node-lib and documented in that library.